woo-razorpay.php

```diff
@@ -596,7 +708,21 @@ private function getRedirectUrl($orderId)
          */
         protected function getRazorpayPaymentParams($orderId)
         {
-            
+            $getWebhookFlag =  get_option('webhook_enable_flag');
```

Comment:
@abdulwahidsharief , is this auto enable again needs to added in this ?

Reply:
@ramth05 no after git conflict resolved it is displaying as new line